server/src/repositories/student.repository.ts

```diff
@@ -248,7 +249,7 @@ export class StudentRepository extends AbstractRepository<Student> {
           isCourseCompleted,
           totalScore,
           certificateId,
-          position: null,
+          position: student_rank,
```

Comment:
let's try to avoid potential confusion and use `rank` as word for student's position in the score